spec/index.bs

```diff
@@ -410,6 +410,15 @@ The [=subresources=] of textures included in the views provided to
 are considered to have {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
 for the [=usage scope=] of this render pass.
 
+The <dfn dfn>physical size</dfn> of a {{GPUTexture}} [=subresource=] is the dimension of the {{GPUTexture}}
+[=subresource=] in pixels that includes the possible extra paddings to form complete [=texel blocks=] in the
+[=subresource=]. The extra paddings only exists in a compressed texture mipmap whose {{GPUTexture/[[textureSize]]}}
+at [=mipmap level=] 0 are multiple of [=texel block size=], but its subdivided levels are not. For example,
```

Comment:
> The extra paddings only exists in a compressed texture mipmap whose {{GPUTexture/[[textureSize]]}}
at [=mipmap level=] 0 are multiple of [=texel block size=], but its subdivided levels are not.

This seems to be a conditional statement, that says that padding exists only if level 0 is a multiple of the texel block size. How about a more explicit explanation like:

Compressed textures always have a mipmap level 0 whose extent is a multiple of the texel block size, but lower mipmap levels might not be multiple of the texel block size and can have padding.

Reply:
Done